### ISSUE TITLE
Where is the documentation

### ISSUE THREAD

**Issue by FabienDehopre** [angular-seattle/blue-harvest#3]
Hello,
First of all, the helpers are really great.
But, do you plan to write a documentation on how to use these helpers in end-to-end tests?

**Comment by kenisteward** [comment]
@FabienDehopre I'm not sure if you've seen their video but it is here

https://www.youtube.com/watch?v=6aPfHrSl0Qk

**Comment by FabienDehopre** [comment]
Of course, I've seen their video about end-to-end tests and blue-harvest at ng-conf18. But we are using blue-harvest at my current job (to make e2e tests easier to write and read) and my management wanted me to write a doc since their is no official written doc.
Now, we have a non-official documentation in our wiki (but it's in french). Maybe I will send a PR with the documentation but it probably won't be 100% correct since I wrote ours by deduction and after seeing the video.

**Comment by andrei-ilyukovich** [comment]
Any updates on documentation?

**Comment by tbowmo** [comment]
found the demo project usefull https://github.com/angular-seattle/protractor-new-hope

**Comment by heathkit** [comment]
I'm glad you find blue-harvest useful! This is based on an internal library, which we plan to open source in the next six months or so as a more complete project. We wanted to demonstrate the action helpers style of writing e2e tests for ng-conf, so we released blue-harvest as kind of a preview. 

Any documentation PRs would be welcome! We're happy to review and suggest corrections :)

**Comment by tbowmo** [comment]
@heathkit oh no.. Don't hold us in suspense for the next 6 months.. Specially when I finally got around to implement blue-harvest screen comparison into our smoketest.. :)
